lib/options.json

```diff
@@ -157,27 +137,21 @@
       ]
     },
     "pfxPassphrase": {
-      "description": "The passphrase to a (SSL) PFX file.",
       "type": "string"
     },
     "requestCert": {
-      "description": "Enables request for client certificate. This is passed directly to the https server.",
       "type": "boolean"
     },
     "inline": {
-      "description": "Enable inline mode to include client scripts in bundle (CLI-only).",
       "type": "boolean"
     },
     "disableHostCheck": {
-      "description": "Disable the Host header check (Security).",
       "type": "boolean"
     },
     "public": {
-      "description": "The public hostname/ip address of the server.",
       "type": "string"
     },
     "https": {
-      "description": "Enable HTTPS for server.",
```

Comment:
Don't remove description, In the future, we can derive more friendly errors based on them